pkgs/tools/security/wordlists/default.nix

```diff
@@ -0,0 +1,78 @@
+{ lib
+, runtimeShell
+, stdenv
+, tree
+, pkgs
+}:
+
+let
+  _wordlistData =
+    [ { name = "dirbuster"; expressionPath = ./dirbuster.nix; }
+      { name = "nmap"; expressionPath = ./nmap.nix; needsPackage = true; }
+      { name = "rockyou"; expressionPath = ./rockyou.nix; }
+      { name = "seclists"; expressionPath = ./seclists.nix; }
+      { name = "wfuzz"; expressionPath = ./wfuzz.nix; needsPackage = true; }
+    ];
+
+  scopedWordlists = lib.makeScope pkgs.newScope (self: with self;
+    let _packageWithLinkCommandFrom = acc: datum:
+      # Some wordlists are based on an already existing `nixpkgs` package; Pass it as parameter of the expression.
+      let parameters = {}
+        // lib.optionalAttrs ((builtins.elem "needsPackage" (builtins.attrNames datum)) && datum.needsPackage) { ${datum.name} = pkgs.${datum.name}; };
+      in
+        let p = callPackage datum.expressionPath parameters;
+        in acc // { ${datum.name} = p; };
+    in builtins.foldl' _packageWithLinkCommandFrom {} _wordlistData
+  );
+in let
+  # Given an array of wordlist packages,
+  # return an array of bash commands to create symlinks to the files shared by each package.
+  allLinksCommandFor = wordlistPackages:
```

Comment:
I think you could use [`symlinkJoin`](https://github.com/NixOS/nixpkgs/blob/6af93b840564dd307f92d237569bad947ebf25e2/pkgs/build-support/trivial-builders.nix#L282) here instead ([usage description](https://nixos.wiki/wiki/Nix_Cookbook#Wrapping_packages))